vavr/src-gen/main/java/io/vavr/Tuple0.java

```diff
@@ -95,6 +95,11 @@ public int compareTo(Tuple0 that) {
         return List.empty();
     }
 
+    public <T1> Tuple1<T1> concat(T1 t1) {
+        Objects.requireNonNull(t1, "t1 is null");
```

Comment:
Do we need this check here?

Reply:
Yeah I guess you are right. I'll remove those.